Makefile

```diff
@@ -14,8 +14,6 @@ OS_TYPE ?= $(shell uname)
 # Module Registry used for pushing the image
 MODULE_REGISTRY_PORT ?= 8888
 MODULE_REGISTRY ?= op-kcp-registry.localhost:$(MODULE_REGISTRY_PORT)/unsigned
```

Comment:
These stay? Because they are not specific to the `module-template` but the image?

Reply:
Yes.